SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -367,6 +368,12 @@ void ASpatialFunctionalTest::FinishTest(EFunctionalTestResult TestResult, const
 	}
 }
 
+void ASpatialFunctionalTest::AddExpectedLogError(const FString& ExpectedPatternString, int32 Occurrences /*= 1*/,
+												 bool ExactMatch /*= false*/)
```

Comment:
bExactMatch

Reply:
It was a copy of the API defined by the blueprint library, but I'll change it for ours.